workspaces/marketplace/plugins/marketplace-common/src/types.ts

```diff
@@ -96,6 +118,39 @@ export interface MarketplacePluginSpec extends JsonObject {
   };
 }
 
+
+/**
+ * @public
+ */
+export type FullTextFilter = {
+  term: string;
+  fields?: string[];
+};
+
+/**
+ * @public
+ */
+export type GetPluginsRequest = {
+  limit?: number;
+  offset?: number;
+  filter?: Record<string, string>;
+  orderFields?: EntityOrderQuery;
+  searchTerm?: string;
+};
+
+/**
+ * @public
+ */
+export interface MarketplaceApi {
+  getPlugins(
+    request?: GetPluginsRequest,
+  ): Promise<MarketplacePluginWithPageInfo>;
+  getPluginByName(name: string): Promise<MarketplacePlugin>;
+  getPluginLists(): Promise<MarketplacePluginList[]>;
+  getPluginListByName(name: string): Promise<MarketplacePluginList>;
+  getPluginsByPluginListName(name: string): Promise<MarketplacePlugin[]>;
+}
```

Comment:
Where was the old MarketplaceApi interface defined???

Reply:
Its down in the end of this file :) Probably this is repeated after merge conflict.